api_tests/nodes/views/test_node_files_list.py

```diff
@@ -358,6 +358,16 @@ def test_node_files_are_filterable_by_kind(self):
         assert_equal(len(res.json['data']), 1)  # filters out 'xyz'
         assert_equal(res.json['data'][0]['attributes']['name'], 'abc')
 
+    def test_node_files_are_filterable_by_size(self):
+        self.file = api_utils.create_test_file_for_size_filter(self.project, self.user)
+
+        url = '/{}nodes/{}/files/osfstorage/?filter[size]=123'.format(API_BASE, self.project._id,)
+        res = self.app.get(url, auth=self.user.auth)
+        print(res)
+        assert_equal(res.status_code, 200)
+        assert_equal(res.json['data'][0]['attributes']['name'], 'test_file')
+        assert_equal(res.json['data'][0]['attributes']['size'], 123)
```

Comment:
Can you get the size off of self.file directly rather than using `123`? The problem is that someone could change the size in the helper function and then the test would fail.